### PR TITLE
Minor UI Changes

### PR DESCRIPTION
**Pull request by hbafna1**
Home Page UI Changes, 1 Image added.

### CODE REVIEW

File: web/doctor/templates/doctor/patient_reports.html [hunk 1]
```diff
@@ -43,7 +43,9 @@ <h1> Patient Reports </h1>
     <p>File Name: {{i.1}}</p>
     <p>Date and Time: {{i.0}}</p>
 
-    <a href="https://ipfs.infura.io/ipfs/{{i.2}}" class="btn btn-info mx-auto" target="_blank">View Report</a>
+    <a onclick="myFunction()" href="https://ipfs.infura.io/ipfs/{{i.2}}" class="btn btn-info mx-auto" target="_blank">View Report</a>
```
Comment (amey-balekundri) [inline web/doctor/templates/doctor/patient_reports.html:46]: remove onclick function

File: web/doctor/templates/doctor/patient_reports.html [hunk 2]
```diff
@@ -52,5 +54,10 @@ <h1> Patient Reports </h1>
 
 <p id='result'></p>
 
+<script>
```
Comment (amey-balekundri) [inline web/doctor/templates/doctor/patient_reports.html:57]: remove this function

**Review by amey-balekundri — CHANGES_REQUESTED** [review]

**Review by hbafna1 — COMMENTED** [review]
Changes Done.

**Comment by amey-balekundri** [comment]
Merged

**Pull request merged**